utilize cra's pwa functionality

johnrackles opened the pull request:
fixes #11 

Or is there anything specific you want to achieve that goes beyond CRA's PWA capabilities?

ericadamski commented:
Thats great! I don't know about anything past CRA's PWA stuff, what is there to be done?

johnrackles commented:
I think for your usecase it's enough. I've added an icon and updated the manifest file so it uses the correct app name when you save the app on your homescreen.
